src/lib/routes/admin-api/user-admin.ts

```diff
@@ -175,7 +204,22 @@ export default class UserAdminController extends Controller {
                 openApiService.validPath({
                     tags: ['Users'],
                     operationId: 'searchUsers',
-                    responses: { 200: createResponseSchema('usersSchema') },
+                    summary: 'Search users',
+                    description:
+                        ' It will preform a simple search based on name and email matching the given query. Requires minimum 2 characters',
+                    parameters: [
+                        {
+                            name: 'q',
+                            description:
+                                'The pattern to search in the username or email',
+                            schema: { type: 'string' },
+                            in: 'query',
+                        },
+                    ],
```

Comment:
If you need to type this somehow (you might not), then you could look into using the `FromQueryParams` method and putting it in its own file (similar to how we do it in `src/lib/openapi/spec/export-query-parameters.ts`). I'm not sure whether that's necessary here. As long as it shows up correctly in the output schema, I'm happy 😄

Reply:
I copied this pattern from other places where it was done inline. It does show up in the swagger UI and works fine :)